sitewit/__init__.py

```diff
@@ -1,4 +1,4 @@
 """Python client for the SiteWit API."""
 
-__version__ = '0.8.0'
+__version__ = '0.9.0'
```

Comment:
That applies to libs which aren't under development.  For libs under dev (when major version < 1) the rules are different.  And we came into agreement that to be able to distinguish between backwards compatible and non-compatible changes we should increment minor for 1st case and patch for the second.